tests/test_flow/test_reload.py

```diff
@@ -0,0 +1,257 @@
+import pytest
+
+import sys
+from textwrap import dedent
+
+from bionic.exception import CodeVersioningError
+
+
+# Pytest rewrites modules and caches the compiled pyc file. The cache can
+# interfere with the tests below. Hence, we disable the cache here.
+@pytest.fixture(autouse=True)
+def dont_write_bytecode():
+    dont_write_bytecode = sys.dont_write_bytecode
+    yield
+    sys.dont_write_bytecode = dont_write_bytecode
+
+
+@pytest.fixture
+def tmp_module_path(tmp_path):
+    tmp_dir_name = str(tmp_path)
+    sys.path.insert(0, tmp_dir_name)
+
+    sys.dont_write_bytecode = True
```

Comment:
Is this intentional?

Reply:
Oops, this is meant to be part of `dont_write_bytecode()`